packages/editor/src/components/post-publish-panel/postpublish.js

```diff
@@ -48,24 +49,32 @@ class PostPublishPanelPostpublish extends Component {
 	}
 
 	render() {
-		const { children, isScheduled, post, postType } = this.props;
+		const { children, isScheduled, post, postType, instanceId } = this.props;
+		const postLabel = get( postType, [ 'labels', 'singular_name' ] );
 		const viewPostLabel = get( postType, [ 'labels', 'view_item' ] );
+		const id = `post-publish-panel__postpublish-link-input-${ instanceId }`;
 
 		const postPublishNonLinkHeader = isScheduled ?
 			<Fragment>{ __( 'is now scheduled. It will go live on' ) } <PostScheduleLabel />.</Fragment> :
 			__( 'is now live.' );
-		const postPublishBodyText = isScheduled ?
-			__( 'The post address will be:' ) :
-			__( 'What’s next?' );
 
 		return (
 			<div className="post-publish-panel__postpublish">
 				<PanelBody className="post-publish-panel__postpublish-header">
 					<a href={ post.link }>{ post.title || __( '(no title)' ) }</a> { postPublishNonLinkHeader }
 				</PanelBody>
 				<PanelBody>
-					<div><strong>{ postPublishBodyText }</strong></div>
+					<p className="post-publish-panel__postpublish-subheader">
+						<strong>{ __( 'What’s next?' ) }</strong>
+					</p>
+					<label htmlFor={ id } className="post-publish-panel__postpublish-link-label">
```

Comment:
Can we use [Text Control ](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/text-control) component to simplify this logic? (it automatically generates the ids).